src/Tizen.Multimedia/MediaTool/MediaPacket.cs

```diff
@@ -170,7 +170,7 @@ public MediaFormat Format
         }
 
         /// <summary>
-        /// Gets or sets the PTS(Presentation Time Stamp) value of the current packet.
+        /// Gets or sets the PTS(Presentation Time Stamp) value in nanoseconds of the current packet.
```

Comment:
I didn't guide you like above.

I left a comment on src/Tizen.Multimedia/AudioManager/AudioDucking.cs as follow.
```
journey2w 3 days ago
Suggestion:

    /// <param name="duration">The duration for ducking in milliseconds</param>
```

If you write the sentence, the meaning is changed because of **'nanoseconds of the current packet'**.
I think the following sentence better.

/// Gets or sets the PTS(Presentation Time Stamp) value of the current packet in nanoseconds.